packages/make-styles/just.config.ts

```diff
@@ -1,3 +1,4 @@
 import { preset } from '@fluentui/scripts';
 
-preset();
+// need to build commonjs even in min mode because it's used by babel-make-styles
+preset({ alwaysBuildCommonjs: true });
```

Comment:
This change is to fix this issue:
```
verb @fluentui/react-divider build |  [9:49:30 PM] x ------------------------------------
verb @fluentui/react-divider build |  [9:49:30 PM] x Error: [BABEL]: Cannot find module '/tmp/054cb751/node_modules/@fluentui/make-styles/lib-commonjs/index.js'. Please verify that the package.json has a valid "main" entry (While processing: /tmp/054cb751/packages/babel-make-styles/lib/index.js)
verb @fluentui/react-divider build |      at tryPackage (internal/modules/cjs/loader.js:295:19)
verb @fluentui/react-divider build |      at Function.Module._findPath (internal/modules/cjs/loader.js:508:18)
verb @fluentui/react-divider build |      at Function.Module._resolveFilename (internal/modules/cjs/loader.js:802:27)
verb @fluentui/react-divider build |      at Function.Module._load (internal/modules/cjs/loader.js:667:27)
verb @fluentui/react-divider build |      at Module.require (internal/modules/cjs/loader.js:887:19)
verb @fluentui/react-divider build |      at require (internal/modules/cjs/helpers.js:74:18)
verb @fluentui/react-divider build |      at Object.<anonymous> (/tmp/054cb751/packages/babel-make-styles/lib/plugin.js:9:21)
```

Reply:
Never mind it didn't work...I think babel-make-styles needs the commonjs version of all packages, so I updated the PR to always include that step even in the min build